test/graphql/mutations/associate_profiles_test.rb

```diff
@@ -31,7 +31,8 @@ class AssociateProfilesMutationTest < ActiveSupport::TestCase
       context: { current_user: users(:test) }
     )['data']['associateProfiles']['system']
 
-    assert_equal hosts(:one).reload.profiles, [profiles(:one), profiles(:two)]
+    assert_equal Set.new(hosts(:one).reload.profiles),
+                 Set.new([profiles(:one), profiles(:two)])
```

Comment:
This test sometimes fails if the arrays aren't in the same order. We've had to do this for a lot of our tests so that they don't sometimes fail.

Reply:
This is just an idea, but you could `assert hosts(:one).reload.profiles.includes?(profiles([:one, :two].sample))`

Reply:
That would be okay, but it doesn't check that both profiles exist. There's no problem using sets like this that I can think of, and they have no real performance overhead, especially with length 2.